learn/cookbooks/running_production.mdx

```diff
@@ -3,106 +3,106 @@ title: Running Meilisearch in production — Meilisearch documentation
+Next, use `curl` to download and run the Meilisearch command-line installer:
 
-You can always check the latest Meilisearch stable version, and get Meilisearch for the Operating System of your choice, by visiting the following link:
+```sh
+# Install Meilisearch latest version from the script
+curl -L https://install.meilisearch.com | sh
+```
 
-**[Latest Meilisearch Stable Version](https://github.com/meilisearch/meilisearch/releases/latest)**
+The Meilisearch installer is a set of scripts that ensure you will get the correct binary for your system.
 
-Give the binary execute permission using:
+Give the binary the required execute permission:
 
-```bash
+```sh
 chmod +x meilisearch
 ```
 
-Meilisearch is finally installed and ready to use. To make it accessible from everywhere in your system, move the binary file into your system binaries folder:
+The last step is to make the binary accessible from anywhere in your system. Move the binary file into `/usr/local/bin`:
 
-```bash
-# Move the Meilisearch binary to your system binaries
+```sh
 mv ./meilisearch /usr/local/bin/
 ```
 
+Meilisearch is now ready to use. 
+
 ## Step 2: Create system user
 
-Running applications as root can introduce security flaws in your system. To prevent that from happening, create a dedicated system user for running Meilisearch:
+Running applications as root exposes you to unnecessary security risks. To prevent that, create a dedicated user for Meilisearch:
 
-```bash
+```sh
 useradd -d /var/lib/meilisearch -b /bin/false -m -r meilisearch
 ```
 
 ## Step 3: Create a configuration file
 
-Download default config to `/etc`:
+After installing Meilisearch and taking the first step towards keeping your data safe, you need to set up a basic configuration file.
+
+First, create the directories where Meilisearch will store its data:
+
+```bash
+mkdir /var/lib/meilisearch/data /var/lib/meilisearch/dumps /var/lib/meilisearch/snapshots
+chown -R meilisearch:meilisearch /var/lib/meilisearch
+chmod 750 /var/lib/meilisearch
+```
+
+Next, download the default configuration to `/etc`:
 
 ```bash
 curl https://raw.githubusercontent.com/meilisearch/meilisearch/latest/config.toml > /etc/meilisearch.toml
 ```
 
-Update the following lines so Meilisearch stores its data in the home folder of your newly created user:
+Finally, update the following lines so Meilisearch uses the directories you created earlier to store its data, replacing `MASTER_KEY` with a 16-byte string:
 
 ```ini
 env = "production"
-master_key = "YOUR_MASTER_KEY_VALUE"
+master_key = "MASTER_KEY"
 db_path = "/var/lib/meilisearch/data"
 dump_dir = "/var/lib/meilisearch/dumps"
 snapshot_dir = "/var/lib/meilisearch/snapshots"
 ```
 
-Finally, create the directories you added to the configuration file and set proper privileges:
+Remember to choose a safe master key and avoid exposing it in publicly accessible locations.
```

Comment:
I think here you could let them know about this:
<img width="1033" alt="image" src="https://github.com/meilisearch/documentation/assets/4116980/9289e42d-1a8d-4b9f-a76d-9d3882fba7fb">